system-design/app-interoperability/observability.md

```diff
@@ -69,24 +69,27 @@ For devices running Kubernetes the following is a minimum list of observability
 
 Cluster events observability data MUST be collected. It is recommended the Device Owner use either the [Kubernetes Objects Receiver](https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-objects-receiver) or [Kubernetes Event Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/k8seventsreceiver/README.md) with the default configuration to collect this information but using either of these receivers is not required.
 
 If the Device Owner chooses not to use either the Kubernetes Object Receiver or Kubernetes Event Receiver they MUST provide the same output as these Kubernetes Event Receiver's default configuration.
 > **Action:** Need to determine which namespaces should be included. All of them, or just the ones the device owner is responsible for creating.
 >
-> **Action:** Determine how we want to represent this information and what the defaults should be if the current defaults are not right.
+> **Action:** The Kubernetes objects receiver needs to be configured to export events and other resource logs so we'll need to document something additional for this receiver.
 
 Node, Pod and Container observability data MUST be collected. It is recommended the Device Owner use the [Kubelet Stats Receiver](https://opentelemetry.io/docs/kubernetes/collector/components/#kubeletstats-receiver) with the default configuration to collect this information but using this receiver is not required.
 
 If the Device Owner chooses not to use the Kubelet Stats Receiver they MUST provide the same output as the Kubelet Stats Receiver's default configuration.
-> **Action:** Determine how we want to represent this information and what the defaults should be if the current defaults are not right.
+
+> **Note:** Please see the [information below](#application-observability-default-telemetry) for the default metrics emitted by the Kubelet Stats Receiver.
 
 Metadata identifying the observability data's source MUST be added to the received observability data. It is recommend the Device Owner use the [Kubernetes Attributes Processor](https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-attributes-processor) with the default configuration to enhance the observability data with this additional metadata but using this processor is not required.
 
 If the Device Owner chooses not to use the Kubernetes Attributes Processor they MUST provide the same metadata as the Kubernetes Attributes Processor's default configuration
-> **Action:** Determine how we want to represent this information and what the defaults should be if the current defaults are not right.
+
+> **Note:** Please see the [information below](#application-observability-default-telemetry) for the default attributes added by the Kubernetes Attributes Processor.
 
 **Non-Clustered Container Platforms (e.g. Docker, Podman)**
```

Comment:
I recommend renaming this section to "Standalone Device Container Platforms" and removing the reference to docker/podman in the title. 

Mentioning them in the body seems ok to me.

Reply:
It was also a little unclear with the formatting what was a MUST. It's mentioned the following but tough to follow which sections are required. 
Maybe a bullet list under the paragraph menioning MUST?

Reply:
I reformated the section a little bit to hopefully make it a bit more clear.